pybaseball/datasources/fangraphs.py

```diff
@@ -42,16 +43,55 @@ def __init__(self):
     def _postprocess(self, data: pd.DataFrame) -> pd.DataFrame:
         return data
 
+    def _sort(self, data: pd.DataFrame, columns: List[str], ascending: bool = True) -> pd.DataFrame:
+        known_columns = [x for x in columns if x in data.columns]
+        if known_columns == []:
+            return data
+
+        return data.sort_values(columns, ascending=ascending)
+
     def _validate(self, data: pd.DataFrame) -> pd.DataFrame:
         return data
 
-    def fetch(self, start_season: int, end_season: Optional[int] = None, types: FanGraphsStat = None,
+    def fetch(self, start_season: int, end_season: Optional[int] = None, types: List[FanGraphsStat] = [],
               league: FanGraphsLeague = FanGraphsLeague.ALL, qual: Optional[int] = None, split_seasons: bool = True,
               month: FanGraphsMonth = FanGraphsMonth.ALL, on_active_roster: bool = False, minimum_age: int = MIN_AGE,
               maximum_age: int = MAX_AGE, team: str = '0', _filter: str = '', players: str = '',
               position: FanGraphsPositions = FanGraphsPositions.ALL, max_results: int = 1000000,) -> pd.DataFrame:
 
-        types = self.DEFAULT_TYPES if types is None else types
+        """
+        Get leaderboard data from FanGraphs.
+
+        ARGUMENTS:
+        start_season       : int                : First season to return data for
+        end_season         : int                : Last season to return data for
+                                                  Default = start_season
+        types              : str                : The columns of data to return
+        league             : FanGraphsLeague    : League to return data for: ALL, AL, FL, NL
```

Comment:
I think in these cases the API should be a string (= "AL", "NL" etc (pref not case sensitive) and then we should handle the enum

Reply:
Implemented.